From: SeongJae Park <sj@kernel.org>
Date: Tue, 8 Oct 2024 17:37:34 -0700
Subject: [PATCH] mm/damon/Kconfig: update DBGFS_KUNIT prompt copy for
 SYSFS_KUNIT

CONFIG_DAMON_SYSFS_KUNIT_TEST prompt is copied from that for DAMON
debugfs interface kunit tests, and not correctly updated.  Fix it.

Fixes: b8ee5575f763 ("mm/damon/sysfs-test: add a unit test for damon_sysfs_set_targets()")
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/damon/Kconfig b/mm/damon/Kconfig
index 35b72f88983a..d0357f3e9372 100644
--- a/mm/damon/Kconfig
+++ b/mm/damon/Kconfig
@@ -60,7 +60,7 @@ config DAMON_SYSFS
 	  the interface for arbitrary data access monitoring.
 
 config DAMON_SYSFS_KUNIT_TEST
-	bool "Test for damon debugfs interface" if !KUNIT_ALL_TESTS
+	bool "Test for damon sysfs interface" if !KUNIT_ALL_TESTS
 	depends on DAMON_SYSFS && KUNIT=y
 	default KUNIT_ALL_TESTS
 	help
-- 
2.39.5

